javascript/javascript1/week3/when-will-be-there.js

```diff
@@ -0,0 +1,41 @@
+function calculateTravelTime(travelInfo) {
+  const timeInHours = travelInfo.destinationDistance / travelInfo.speed;
+
```

Comment:
Looks Correct, 

Something to keep in mind for future, what happens if destinationDistance or speed are not postive numbers? 
Might be good to add some checks to ensure speed and destinationDistance are positive numbers :)